### ISSUE TITLE
ARIA12 Not sure why it fails WCAG to use role=heading

### ISSUE THREAD

**Issue by DavidMacDonald** [wet-boew/wet-boew-wpss#100]
The tool failed this code

```  <div class="footer-dropUp-title" role="heading" aria-level="1">CHOOSE YOUR LOCATION</div>```

I don't think this fails WCAG. I don't think its a failure of WCAG to even override the semantics of an existing heading.

**Comment by irwink** [comment]
The thinking behind the check (and error) is that for headings 1-6 the native HTML tag (h1 - h6) should be used.  Headings greater than 6 can use the role and aria-level attributes.  In the ARIA12 description it states "When possible, use native heading mark-up directly.", I just elevated it to an error as not all user agents may fully support ARIA.

**Comment by DavidMacDonald** [comment]
I understand... but I think  I'll comment it out because technically its not a WCAG failure...

**Comment by mgifford** [comment]
This sounds like this issue has been resolved. That correct @DavidMacDonald ?
